apps/src/templates/Button.jsx

```diff
@@ -203,7 +205,8 @@ class Button extends React.Component {
       id,
       tabIndex,
       isPending,
-      pendingText
+      pendingText,
+      useButtonTag
```

Comment:
This is temporary. I'm currently working on deprecating the <div> and <a> version of the Button.